compiler/compiler-settings.cpp

```diff
@@ -312,6 +312,7 @@ void CompilerSettings::init() {
     ss << " -std=c++17";
   #elif __cplusplus <= 202002L
     ss << " -std=c++20";
+    ss << " -Wno-type-limits -Wno-attributes -Wno-ignored-attributes";
```

Comment:
Can we avoid of passing this parameters?

Reply:
sure